Database/Constraint/Checks/Locker Checks.sql

```diff
@@ -9,13 +9,16 @@ GO
 /* Testdata */
```

Comment:
Style ontbreekt.  Er staat bovenaan niet wie deze constraint heeft gemaakt, op welke dag en datum en wat de naam van de business rule is..   

Even de Style aanvullen.